buildSrc/src/main/java/org/elasticsearch/gradle/tasks/Run.java

```diff
@@ -0,0 +1,37 @@
+package org.elasticsearch.gradle.tasks;
+
+import org.elasticsearch.gradle.testclusters.RunTask;
+import org.gradle.api.DefaultTask;
+import org.gradle.api.tasks.Internal;
+import org.gradle.api.tasks.TaskProvider;
+import org.gradle.api.tasks.options.Option;
+
+public class Run extends DefaultTask {
```

Comment:
Man, this whole thing is a goofy hack. It's really just so we can alias `:distribution:run` as simply `run` and still pass along the CLI options.

@rjernst since pretty much everyone just uses `./gradlew :run` why don't we just _move_ that task to the root project rather than creation it in the `:distribution` project and do this weird aliasing nonsense?

Reply:
That's fine I guess. It means pulling in testclusters into the root project.

Reply:
I've changed this and moved all related to this into gradle/run.gradle script and apply it on the root project now

Reply:
>  It means pulling in testclusters into the root project.

I think that's fine, and a net-zero since we can then remove that plugin from the `:distribution` project.